msteams-platform/tabs/what-are-custom-tabs.md

```diff
@@ -6,40 +6,48 @@ ms.topic: overview
+Custom tabs are your hosted Teams-aware IFrame webpages embedded in Microsoft Teams. The dedicated tab experience can be added as part of a [channel or group chat](~/foo.md) inside a Team or as a [personal app](~/foo.md) for an individual user. As part of your app, you can add custom tabs, using our [Teams JavaScript client SDK](/javascript/api/overview/msteams-client), to embed your own web content in Teams and add Teams-specific functionality.
 
-There are two types of tabs available in Teams - group/channel and personal. A group/channel tab delivers content to group chats and channels, and are a great way to create collaboration spaces around web-based content. Personal tabs, along with one-to-one bots, are part of personal apps and are scoped to a single user. They can be pinned to the left navigation bar and are very good at increasing productivity by making your service available directly inside the Teams client.
+There are two types of tabs available in Teams - channel/group and personal. A channel/group tab delivers content to channels and group chats, and are a great way to create collaborative spaces around dedicated web-based content. Personal tabs, along with direct conversation bots, are part of personal apps and are scoped to a single user. They can be pinned to the left navigation bar and promote increased productivity by making your service available directly inside the Teams client.
 
 ## Tabs user scenarios
 
-**Scenario:** Make an existing web-based resource available inside Microsoft Teams.
-**Example:** You have an informational corporate website you make available as a personal tab in Teams.
+**Scenario:** Bring an existing web-based resource inside Teams. \
+**Example:** You create a personal tab in your Teams app that presents an informational corporate website to users.
 
-**Scenario:** You want to add an about or help page for your bot or messaging extension.
-**Example:** You have a static webpage that you make available as a personal tab in Teams.
+**Scenario:** Add static support pages to a Teams bot or messaging extension. \
+**Example:** You create personal tabs that provide about and help webpage content to users.
 
-**Scenario:** You have items in a system that your users need to discuss or collaborate around regularly.
-**Example:** You create a group/channel tab that is configurable with deep linking to individual items.
+**Scenario:** Provide accessibility to system-level items your users access regularly for cooperative dialogue and collaboration. \
+**Example:** You create a configurable channel/group tab with deep linking to individual items.
 
-## How do tabs work?
+## How do tabs function?
```

Comment:
Please use "work". Always default to the simplest possible word/phrase to define a concept. Also all the other pages like this one will have a similarly worded section